tests/v2/actions/observability/config.go

```diff
@@ -5,4 +5,152 @@ type StackStateConfig struct {
 	Url            string `json:"url" yaml:"url"`
 	ClusterApiKey  string `json:"clusterApiKey" yaml:"clusterApiKey"`
 	UpgradeVersion string `json:"upgradeVersion" yaml:"upgradeVersion"`
+	License        string `json:"license" yaml:"license"`
+	AdminPassword  string `json:"adminPassword" yaml:"adminPassword"`
+}
+
+// GlobalConfig represents global configuration values
+type GlobalConfig struct {
+	ImageRegistry string `json:"imageRegistry" yaml:"imageRegistry"`
+}
+
+// AuthenticationConfig represents the authentication configuration
+type AuthenticationConfig struct {
+	AdminPassword string `json:"adminPassword" yaml:"adminPassword"`
+}
+
+// ApiKeyConfig represents API key configuration
+type ApiKeyConfig struct {
+	Key string `json:"key" yaml:"key"`
+}
+
+// LicenseConfig represents the license configuration
+type LicenseConfig struct {
```

Comment:
nit, Since each of the configs contains only a single string field, why not define them directly in StackStateServerConfig as strings instead of separate structs?

Reply:
Originally this was part of a larger struct. This was my attempt to break it down in to smaller chunks. Is there any benefit of creating a larger struct?

Reply:
It seems like there are multiple structseach containing only a single string which felt a bit redundant. Would it make sense to consolidate them into a single struct for simplicity?

Reply:
I'll restore the original struct. Though to me this broken down one seems more manageable